parser/ParseStatement.hpp

```diff
@@ -148,11 +162,20 @@ class ParseStatementCreateTable : public ParseStatement {
     for (const ParseAttributeDefinition& attribute_definition : *attribute_definition_list_) {
       container_child_fields->back().push_back(&attribute_definition);
     }
+
+    if (opt_block_properties_) {
```

Comment:
Shouldn't the value pointed by opt_block_properties_ be checked for nullptr, instead of opt_block_properties_ pointer itself?